src/OpenTelemetry.Exporter.Console/.publicApi/net461/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,9 @@
+OpenTelemetry.Exporter.ConsoleExporter
+OpenTelemetry.Exporter.ConsoleExporter.ConsoleExporter(OpenTelemetry.Exporter.ConsoleExporterOptions options) -> void
+OpenTelemetry.Exporter.ConsoleExporterOptions
+OpenTelemetry.Exporter.ConsoleExporterOptions.ConsoleExporterOptions() -> void
+OpenTelemetry.Exporter.ConsoleExporterOptions.DisplayAsJson.get -> bool
+OpenTelemetry.Exporter.ConsoleExporterOptions.DisplayAsJson.set -> void
+OpenTelemetry.Trace.ConsoleExporterHelperExtensions
+override OpenTelemetry.Exporter.ConsoleExporter.Export(in OpenTelemetry.Batch<System.Diagnostics.Activity> batch) -> OpenTelemetry.ExportResult
+static OpenTelemetry.Trace.ConsoleExporterHelperExtensions.AddConsoleExporter(this OpenTelemetry.Trace.TracerProviderBuilder builder, System.Action<OpenTelemetry.Exporter.ConsoleExporterOptions> configure = null) -> OpenTelemetry.Trace.TracerProviderBuilder
```

Comment:
should this method be in microsoft namespace so it will be suggested with intellisense without the need to add a namespace?

Reply:
nvm